### PR TITLE
Gabe/call places

### PR DESCRIPTION
**Pull request by glstewart17**
Return stores with stats when places is called.

### CODE REVIEW

File: shopsafe-backend/src/main/java/com/google/sps/servlets/StoresServlet.java [hunk 1]
```diff
@@ -38,22 +49,53 @@
 @WebServlet("/stores")
 public class StoresServlet extends HttpServlet {
 
+    public static final String PLACE_URL = "https://maps.googleapis.com/maps/api/place/nearbysearch/json?location=";
+    public static final String PLACE_TYPE = "&radius=12000&type=grocery_or_supermarket";
+    private String PLACE_KEY;
+
     /**
      * For a get request, return all nearby stores world.
```
Comment (caroljli) [inline shopsafe-backend/src/main/java/com/google/sps/servlets/StoresServlet.java]: I'm a little confused by this wording: what does nearby stores world mean?

File: shopsafe-backend/src/main/java/com/google/sps/servlets/StoresServlet.java [hunk 2]
```diff
@@ -38,22 +49,53 @@
...
+        // Todo: Get address from response
+        
         // Todo: Check address and get LatLon
-
-        // Todo: Get all grocery stores based on LatLon
-
-        // Todo: Migrate stores to internal classes
+        LatLng location = new LatLng(40.163249, -76.395991);
+
+        // Get all grocery stores based on LatLon and migrate to StoreNoScore class.
```
Comment (caroljli) [inline shopsafe-backend/src/main/java/com/google/sps/servlets/StoresServlet.java]: LatLng

File: shopsafe-backend/src/main/java/com/google/sps/servlets/StoresServlet.java [hunk 3]
```diff
@@ -62,9 +104,60 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
+        List<StoreNoScore> stores = new ArrayList<>();
+        
+        try {
+
+            // Read response of call to FCC API given lat and lng.
+            URL url = new URL(PLACE_URL + location.getLatitude() + "," + location.getLongitude()+ PLACE_TYPE + PLACE_KEY);
+            BufferedReader reader = new BufferedReader(new InputStreamReader(url.openStream()));
+            
+            // Store response in json, by reading each line.
+            StringBuffer json = new StringBuffer();
```
Comment (caroljli) [inline shopsafe-backend/src/main/java/com/google/sps/servlets/StoresServlet.java]: Is there a reason why you chose StringBuffer over StringBuilder? I'm just wondering haha

**Review by caroljli — APPROVED** [review]

File: shopsafe-backend/src/main/java/com/google/sps/servlets/StoresServlet.java (hunk 2 above)
Comment (glstewart17) [reply shopsafe-backend/src/main/java/com/google/sps/servlets/StoresServlet.java]: Good catch!

File: shopsafe-backend/src/main/java/com/google/sps/servlets/StoresServlet.java (hunk 1 above)
Comment (glstewart17) [reply shopsafe-backend/src/main/java/com/google/sps/servlets/StoresServlet.java]: Oh my, I really don't know why the word world is there.

File: shopsafe-backend/src/main/java/com/google/sps/servlets/StoresServlet.java (hunk 3 above)
Comment (glstewart17) [reply shopsafe-backend/src/main/java/com/google/sps/servlets/StoresServlet.java]: Not really, they do the same thing, but apparently StringBuilder is slightly faster. So, I made the switch.

**Pull request merged**